scripts/preprocessing/04a-run_ica.py

```diff
@@ -105,15 +105,20 @@ def make_epochs_for_ica(raw, subject, session):
                             root=config.deriv_root,
                             check=False)
     epochs = mne.read_epochs(epochs_fname)
-    selection = epochs.selection
+
+    # Not sure why it is necessary to recreate epochs when we already have them
+    if config.no_epoching:
```

Comment:
The base code recreates an epoch object, but I don't understand why. Here I just reuse the epochs created last step, not sure if that's okay

Reply:
> but I don't understand why.

Because we filter the raw data again with a high-pass filter suitable for ICA. We then create new epochs from the newly-filtered data. You'd have to do the same for the "no epoching" case

Reply:
I completely missed that, thanks